src/Pact/Types/ExpParser.hs

```diff
@@ -120,8 +120,7 @@ mkEmptyInfo e = Info (Just (mempty,e))
 
 {-# INLINE mkStringInfo #-}
 mkStringInfo :: String -> MkInfo
-mkStringInfo s d = Info (Just (fromString $ take (_pLength d) $
-                               drop (fromIntegral $ TF.bytes d) s,d))
+mkStringInfo = mkTextInfo . T.pack
```

Comment:
I have a hard time convincing myself that this is just a linting change. How have we tested that this has the same behavior as what it replaces?

Reply:
oh, the listing for `mkStringInfo` and `mkTextInfo` didn't make it into this PR -- since it is part of another commit in #1197. Let me include that here, to make it more obvious.

Btw. this function is already buggy: note how it uses `TF.bytes` as offset but drops unicode `Char`s. So, I hesitate to introduce unit test to test an obviously wrong behavior. #1197 deals with this and related issues.

Reply:
Using the definition of `mkTextInfo` in the following lines, correctness of the change is witnessed by `(T.take a . T.drop b . T.pack) == (fromString . take a . drop b)`